static/templates/actors/character/tabs/biography.hbs

```diff
@@ -15,6 +15,17 @@
                 </div>
             </div>
         </section>
+
+        <section class="appearance-details{{#if editable}} editable{{/if}}">
```

Comment:
This shouldn't be an independent `section` but under the appearance one

Reply:
Converted to grid then so that it fits nicely.